extensions/chocolatey-azuredatastudio.extension/chocolatey-azuredatastudio.extension.nuspec

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>chocolatey-azuredatastudio.extension</id>
+    <version>1.0.0</version>
+    <title>Chocolatey Azure Data Studio servicing extension</title>
+    <summary>Helper functions useful for developing packages for installing Azure Data Studio extensions.</summary>
+    <authors>chocolatey</authors>
+    <owners>chocolatey, pascalberger</owners>
+    <description>
+This package provides helper functions useful for developing packages for installing Azure Data Studio extensions.
+These functions may be used in Chocolatey install/uninstall scripts by declaring this package a dependency in your package's nuspec.
+    </description>
+    <tags>chocolatey extension azure-data-studio admin</tags>
```

Comment:
You are explicitly telling `Start-ChocolateyProcessAsAdmin` not to elevate during installation and there is no need for the admin tag then.

Reply:
please also add `foss` as a tag.

Reply:
Removed `admin` and added `foss` tag